javascript/javascript2/week2/Week2HW.js

```diff
@@ -0,0 +1,126 @@
+"use strict";
+
+//Doubling of number
+
+let numbers = [1, 2, 3, 4];
+let oddNumbers = numbers
+  .filter((newNumbers) => {
+    return newNumbers % 2 !== 0;
+  })
+  .map((newNumbers) => newNumbers * 2);
+console.log(oddNumbers);
+
+//************************************* */
+
+//Working with movies
+
+// Creating an array for short titled movies ( here I defined short title to be less than 10)
+const shortTitledMovies = movies.filter((movie) => movie.title.length < 10);
+console.log(shortTitledMovies);
+
+// Creating an array for long titled movies ( here I defined long title to be more than and equal to 10)
+const longTitledMovies = movies.filter((movie) => movie.title.length >= 10);
+console.log(longTitledMovies);
+
+// finding the number of movies between 1980-1989 using filter method
+const eightyMovies = movies.filter(
+  (movies) => movies.year >= 1980 && movies.year <= 1989
+);
+console.log(`We have ${eightyMovies.length} movies which are produced in 80s.`);
+
+// finding the number of movies between 1980-1989 using foreach method
+// let moviesCount = 0;
+// movies.forEach((movie) => {
+//   moviesCount =
+//     movie.year >= 1980 && movie.year <= 1989 ? moviesCount + 1 : moviesCount;
+// });
+// console.log(moviesCount);
```

Comment:
please remove the commented line of code

Reply:
thank you Aijaz , but I commented them out because its second method and I didnt want it to be repeated :)